changelog.d/443.feature

```diff
@@ -0,0 +1 @@
+Enable allow-listing of specific IP ranges for OpenAPI requests without having to edit disallowedIpRanges.
```

Comment:
```suggestion
Enable allow-listing of specific IP ranges for OpenAPI requests (when using the Provisioner API) without having to edit disallowedIpRanges. This is done via the new `allowedIpRanges` flag. 
```

Reply:
Ah you merged this too quickly for me to accept this suggestion :)